manifest.json

```diff
@@ -41,5 +41,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
```

Comment:
According to https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/blob/315403c38d3520f96b4f2f067eda285725cb8826/index.js#L16, I think this should be `"ember-source": "3.10.0-alpha.1"`